src/m365/spo/commands/page/Page.ts

```diff
@@ -80,10 +80,6 @@ export class Page {
 
   public static save(name: string, webUrl: string, canvasContent: any, logger: Logger, debug: boolean, verbose: boolean): Promise<void> {
     return new Promise<void>((resolve: () => void, reject: (error: any) => void): void => {
-      if (verbose) {
-        logger.log(`Saving ${name} page...`);
```

Comment:
Rather than removing these statements, we should change them to `logger.logToStderr`

Reply:
Fair point, but as the method is not being used, I removed these lines. To have 100% coverage and later to phase out the file.

Reply:
If the method is no longer used, let's remove it altogether 😁

Reply:
That means the tests also need to be updated from the `page.spec.ts`. So thought it would be easier to do it this way, and once the refactoring happens, both the `page.ts` and its specs can be removed at once.

Reply:
If we changed the implementation and we no longer use this method, shouldn't that be reflected in our tests as well?

Reply:
The `page.ts` file has its own spec file and there is more logic in that `page.ts` file. IMO it does make more sense to do a bigger clean-up of the file, than only this method.

Reply:
Agreed, but if we already know that the method is not being used, why keep it in there?

Reply:
As a memory for the good old times. 

PS: Removed it.